flare/gp_algebra.py

```diff
@@ -305,7 +299,6 @@ def get_neg_like_grad(hyps: np.ndarray, training_data: list,
         get_like_grad_from_mats(ky_mat, hyp_mat, training_labels_np)
 
     if output is not None:
-        output.write_to_log('like grad: ' + str(like_grad)+'\n', name="hyps")
-        output.write_to_log('like: ' + str(like)+'\n', name="hyps")
+        output.write_hyps(None, hyps, None, like, like_grad, name="hyps")
```

Comment:
No hyperparameter labels passed here?

Reply:
I guess that's the way we've been doing it but it might be useful.

Reply:
the label is not available in the function and I didn't want to change the argument for the function